### PR TITLE
Added v2 of pihole kit

### PR DESCRIPTION
**Pull request by mike-wade-gravwell**
## Description
 Version 2 of the pihole app adds the functionality to pull data from Gravwell.

## Kits
[pihole](https://github.com/gravwell/kits/tree/main/pihole)

## New Features 
* Added script to pull data directly from a pihole instance into gravwell
* Added three macros to set the script variables

## Changes/Fixes
* Replaced two resources with updated versions
* Replaced lookups in `$PIHOLE_DNS_QUERY` with new resources
* Updated dashboard to use macro query
* Updated README with more detail

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)

### CODE REVIEW

**Comment by mike-wade-gravwell** [comment]
@david-fritz-gravwell I'm going to remove the img reference in the pihole readme and the img directory

File: pihole/MANIFEST [hunk 1]
```diff
@@ -28,30 +28,20 @@
...
-		{
-			"Name": "pihole_rcodes",
-			"Type": 1,
-			"Hash": "0000000000000000000000000000000000000000000000000000000000000000"
-		},
 		{
 			"Name": "pihole_status_codes",
 			"Type": 1,
-			"Hash": "0000000000000000000000000000000000000000000000000000000000000000"
+			"Hash": "fa3a39987aab09e29029ceba10d1d1d48880eb2f449dc3b2748c329696e95382"
```
Comment (kris-watts-gravwell) [inline pihole/MANIFEST:34]: Please kill the hashes when submitting PRs
```
sed -i 's/"Hash": ".*"/"Hash": "0000000000000000000000000000000000000000000000000000000000000000"/g' MANIFEST
```

**Review by kris-watts-gravwell — DISMISSED** [review]
LGTM

File: pihole/MANIFEST (hunk 1 above)
Comment (mike-wade-gravwell) [reply pihole/MANIFEST]: @kris-watts-gravwell zeroed out hashes and added the requested tag macro

**Review by kris-watts-gravwell — APPROVED** [review]
LGTM

**Pull request merged**